### PR TITLE
update NEWS.md and prepare breaking release

### PR DESCRIPTION
**Pull request by SimonDanisch**

### CODE REVIEW

**Comment by SimonDanisch** [comment]
I'll add some more docs for the scene refactor.
@piever, @ffreyer @jkrumbiegel, feel free to push changes to the NEWS.md!

File: NEWS.md [hunk 1]
```diff
@@ -3,6 +3,17 @@
...
+
+- simplify plotting barplot by group  [#1223](https://github.com/JuliaPlots/Makie.jl/pull/1223). This allows specifying x_distance in a bar plot. It corresponds to the bar width + the x_gap. The rationale is that we compute this from the data, assuming that bar width plus x_gap should equal minimum(diffs(x)), but in categorical data we just want that to be 1 (otherwise things get problematic if in some group not all categories are present).
+
+- add logical expressions for ispressed [#1222](https://github.com/JuliaPlots/Makie.jl/pull/1222). Allows for `ispressed(scene, Mouse.left & (Keyboard.left_control | Keyboard.right_control))`
+
+- add depth shift [#1382](https://github.com/JuliaPlots/Makie.jl/pull/1382)
+
+- **Breaking** clean up Scene type [#1192](https://github.com/JuliaPlots/Makie.jl/pull/1192). Long story short, Scene() doesn't create any axes or limits anymore. All keywords like `raw`, `show_axis` have been removed. A scene is now always works like when using the deprecated `raw=true`. All the high level functionality like showing an axis and adding a 3d camera has been moved to `LScene`. See the new `Scene` docs for more info:
```
Comment (mschauer) [inline NEWS.md:16]:
```suggestion
- **Breaking** clean up Scene type [#1192](https://github.com/JuliaPlots/Makie.jl/pull/1192). Long story short, Scene() doesn't create any axes or limits anymore. All keywords like `raw`, `show_axis` have been removed. A scene now always works like when using the deprecated `raw=true`. All the high level functionality like showing an axis and adding a 3d camera has been moved to `LScene`. See the new `Scene` docs for more info:
```

**Comment by ffreyer** [comment]
I added a few new entries and sorted things by pr number

**Comment by ffreyer** [comment]
closes #1391

**Comment by SimonDanisch** [comment]
Alright, this is maybe not 100% polished, but I'm going to finally merge this! :D 
I will need to spend some time on taking a thorough look at docs and regressions after merging a few more PRs anyways.

**Pull request merged**